### ISSUE TITLE
Feature extraction to obtain vectors

### ISSUE THREAD

**Issue by AntonioVispi** [clovaai/generative-evaluation-prdc#9]
Dear Sir, 
it seems that in your work there aren't tools for getting the vectors of features from images. Thus, have you got any advice for me to obtain these features vectors from my real-fake images dataset? 
Thank you so much!

**Comment by hellog2n** [comment]
Hi, I think that it is up to you because you can use any feature extractors for utilizing this metric. 

Commonly, people use InceptionV3(Tensorflow or Pytorch pretrained model) or Vgg16 ImageNet pretrained classifiers for extracting feature vectors and using perceptual evaluation metrics. Although these authors used vgg in their paper, I have used InceptionV3  and this metric has performed well either. I think that people usually extract features from  (2048dim) averge pooling layer of InceptionV3 or (4096 dim) second fully connected layer of VGG16.

You would better load your real and fake images, and then preprocess for extracting feature vectors from these classifiers. You can also manage to obtain [some reference source code line such as FID](https://github.com/mseitzer/pytorch-fid/blob/master/src/pytorch_fid/fid_score.py) about getting feature vectors.

**Comment by AntonioVispi** [comment]
Thank you so much, you were so kind.
Have  a good day!
